src/content/lib/CreateJobContent/CreateJobForm.js

```diff
@@ -35,8 +36,56 @@ class CreateJobForm extends React.Component {
   handleChangeHourlyRating = (event, index, value) => this.setState({ hourlyRatingValue: value })
   // handleChangeState = (event, index, value) => this.setState({ stateValue: value })
 
+  handleBack () {
+    Router.pushRoute('/opportunities')
+  }
```

Comment:
why in create-job we back in opportunitites?
opportunitites is for worker
we must back to dashboard